### PR TITLE
fix postal address

### PR DESCRIPTION
**Pull request by nitro-neal**

### CODE REVIEW

**Review by decentralgabe — APPROVED** [review]

File: credential/exchange/builder_test.go [hunk 1]
```diff
@@ -94,7 +95,60 @@ func TestPresentationDefinitionBuilder(t *testing.T) {
...
+					},
+				},
+			},
+		},
+	})
+
+	assert.NoError(t, err)
+	definition, err := builder.Build()
+	assert.NoError(t, err)
+	println(util.PrettyJSON(definition))
```
Comment (andresuribe87) [inline credential/exchange/builder_test.go:147]: Is this println intentional?

Comment (decentralgabe) [inline credential/exchange/builder_test.go:147]:
```suggestion
```

Comment (decentralgabe) [reply credential/exchange/builder_test.go]: good catch - removed for Neal

File: credential/exchange/builder_test.go [hunk 2]
```diff
@@ -94,7 +95,59 @@ func TestPresentationDefinitionBuilder(t *testing.T) {
...
+						},
+					},
+				},
+			},
+		},
+	})
+
+	assert.NoError(t, err)
+	definition, err := builder.Build()
+	assert.NoError(t, err)
```
Comment (decentralgabe) [inline credential/exchange/builder_test.go:146]:
```suggestion
	assert.NoError(t, err)
	assert.NotEmpty(t, definition)
```

**Pull request merged**